### ISSUE TITLE
pre trained model

### ISSUE THREAD

**Issue by fouad-tech** [CODEJIN/GST_Tacotron#6]
When I try to load the the pretrained weights I get the following Warnings. I tested the model on the provided example in Inference section of README.md and compared the spectrograms, the alignments and the stop tokens with the ones in your Result section, the overall shape is similar but there is some differences. So I think these Warnings are responsible for the differences. Do you have any idea how can these warnings be resolved ?
   

WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.embeddings.Embedding object at 0x7f22d8be95f8> and <tensorflow.python.keras.layers.convolutional.Conv1D object at 0x7f22d8be9a90>).
WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.convolutional.Conv1D object at 0x7f22d8be9a90> and <tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 0x7f2334790c18>).
WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 0x7f2334790c18> and <tensorflow.python.keras.layers.advanced_activations.ReLU object at 0x7f22d2d562e8>).
WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.convolutional.Conv1D object at 0x7f22d2d569b0> and <tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 0x7f22d2d11518>).
WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 0x7f22d2d11518> and <tensorflow.python.keras.layers.advanced_activations.ReLU object at 0x7f22d2d11c18>).
WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.convolutional.Conv1D object at 0x7f22d2d11cc0> and <tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 0x7f22d2d292b0>).
WARNING:tensorflow:Inconsistent references when loading the checkpoint into this object graph. Either the Trackable object references in the Python program have changed in an incompatible way, or the checkpoint was generated in an incompatible program.

Two checkpoint references resolved to different objects (<tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 0x7f22d2d292b0> and <tensorflow.python.keras.layers.advanced_activations.ReLU object at 0x7f22d2d29e48>).

**Comment by CODEJIN** [comment]
Hi, the shape difference of inference mels are because of the dropout in prenet. I think you try to generate the mels one more time, the shape may be different from the previous one. This method is based on the Tacotron 2 paper.

About checkpoint difference problem, I am not sure the clear reason yet.  Although I want to check and fix it, I have no time for it now... I will takes some time.

**Comment by wlzrf2016** [comment]
Hello, I want to use your pre training model to try, but after downloading and decompressing, I found that it can't be used as usual. Pt file. I want to ask how to use it
